File: client-desktop/model/TicketResponseDTO.cs
namespace model {
    public class TicketResponseDTO {
        public long Id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public DateTime DataCriacao { get; set; }
        public string Prioridade { get; set; }
        public string Status { get; set; }
        public string Resposta { get; set; }

        public long ClienteId { get; set; }
        public string NomeCliente { get; set; }

        public long? TecnicoId { get; set; } 
        public string NomeTecnico { get; set; }
    }
}

File: backend/omegatechapi/src/main/java/com/luizcarlos/omegatechapi/model/dto/TicketResponseDTO.java
package com.luizcarlos.omegatechapi.model.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class TicketResponseDTO {
    private Long id;
    private String titulo;
    private String descricao;
    private LocalDate dataCriacao;
    private String prioridade;
    private String status;
    private String resposta;

    private Long clienteId;
    private String nomeCliente;

    private Long tecnicoId;
    private String nomeTecnico;
}
